File: library/src/main/java/com/kennyc/bottomsheet/adapters/ViewHolder.java
package com.kennyc.bottomsheet.adapters;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.kennyc.bottomsheet.R;

public class ViewHolder {
    public TextView title;

    public ImageView icon;

    public ViewHolder(View view) {
        title = view.findViewById(R.id.title);
        icon = view.findViewById(R.id.icon);
        view.setTag(this);
    }
}

File: library/src/main/java/com/kennyc/bottomsheet/adapters/ViewHolder.kt
package com.kennyc.bottomsheet.adapters

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import com.kennyc.bottomsheet.R

internal class ViewHolder(val view: View) {
    val title: TextView = view.findViewById(R.id.title)

    val icon: ImageView = view.findViewById(R.id.icon)
}